recipes/mss/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: mss
+  version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
+    - lxml >=3.6.0
+    - matplotlib >=1.5.1
+    - netcdf4 >=1.2.2
+    - numpy >=1.11.1
+    - paste >=1.7.5.1
+    - pillow >=3.2.0
+    - pyqt >=4.11.4,<5
+    - python >=2.7.12,<3
+    - python-dateutil >=2.5.3
+    - pytz >=2016.4
+    - qt >=4.8.7,<5
+    - requests >=2.10.0
+    - scipy >=0.17.1
+    - sip >=4.16.9
+    - six >=1.10.0
+    - tk >=8.5.18
+  run:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
+    - lxml >=3.6.0
+    - matplotlib >=1.5.1
+    - netcdf4 >=1.2.2
+    - numpy >=1.11.1
+    - paste >=1.7.5.1
+    - pillow >=3.2.0
+    - pyqt >=4.11.4,<5
+    - python >=2.7.12,<3
```

Comment:
Is `Mission Support System`  Python 2 only? You don't have to restrict Python here (See my comment above in the build section.)